rancher/rancher.yaml

```diff
@@ -0,0 +1,91 @@
+
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: cattle-system
+  labels:
+    name: cattle-system
+
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+    name: cattle-admin
+    namespace: cattle-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+    name: cattle-crb
+    namespace: cattle-system
+subjects:
+  - kind: ServiceAccount
+    name: cattle-admin
+    namespace: cattle-system
+roleRef:
+    kind: ClusterRole
+    name: cluster-admin
+    apiGroup: rbac.authorization.k8s.io
+
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+    namespace: cattle-system
+    name: cattle
+spec:
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          app: cattle
+      spec:
+        serviceAccountName: cattle-admin
+        containers:
+        - image: rancher/rancher:stable
+          imagePullPolicy: Always
+          name: cattle-server
+          ports:
+          - containerPort: 80
+            protocol: TCP
+          - containerPort: 443
+            protocol: TCP
+          volumeMounts:
+          - mountPath: /var/lib/rancher
+            name: rancher
+        volumes:
+        - name: rancher
+          hostPath:
+            path: /home/rancher
+---
+apiVersion: v1
+kind: Service
+metadata: 
+  namespace: cattle-system
+  name: cattle-service
+spec: 
+  ports: 
+    - name: http
+      port: 80
+    - name: https
+      port: 443
+  selector: 
+    app: cattle
+---
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: cattle-ingress
+  namespace: cattle-system
+  annotations:
+    kubernetes.io/ingress.class: traefik
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /
+        backend:
+          serviceName: cattle-service
+          servicePort: http
```

Comment:
ok will separate out ingress in post_Install but without that you won't be able to see the UI

Reply:
But we shouldn't open a management interface to the outside world by default without asking the user. This will get a bit easier when the new firewall changes are done, but for now all of the other close it off by default and give instructions for opening it. 

For example:

https://github.com/civo/kubernetes-marketplace/blob/master/mariadb/post_install.md

Reply:
Sure creating post install for Ingress and removing it from the deployment